dream_layer_backend/dream_layer_backend_utils/api_key_injector.py

```
@@ -46,102 +50,100 @@
+        'LUMA_API_KEY': 'luma'
+    }
+    
+    for env_key, service_name in env_mappings.items():
+        api_key = os.environ.get(env_key)
         if api_key:
-            api_keys[env_key] = api_key
-            # Safely truncate for display without assuming length
-            display_key = api_key[:8] + "..." + api_key[-4:] if len(api_key) > 12 else api_key
-            print(f"[DEBUG] Found {env_key}: {display_key}")
+            api_keys[service_name] = api_key
+            logger.debug(f"Found {env_key}: {display_key}")
```

Ruff (0.12.2)

Undefined name `display_key`

(F821)